.env.example

```diff
@@ -4,6 +4,8 @@ PORT=8000
 BETTER_HASH_SALT=
 USERNAME_ADMIN=
 
+SERVICE_ACCOUNT=
+
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Adding a `SERVICE_ACCOUNT` environment variable is a good practice for managing service account credentials. However, it's important to ensure that this sensitive information is securely managed and not exposed in logs or error messages. Also, remember to update the production environment with this new variable.
```diff
+ SERVICE_ACCOUNT=
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->